pkg/job/local_job.go

```diff
@@ -5,27 +5,28 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/openshift/assisted-service/internal/operators"
+
```

Comment:
I'm seeing the extra line for this import across other files. Do you think it's worth consolidating the import with the others below so that they appear to belong to the same group?

Reply:
:+1: